competition_inputs/BottleJSONReader.py

```diff
@@ -0,0 +1,18 @@
+import json
```

Comment:
This file should be moved under the vision folder. Also, the file name should follow `snake_case`.